h2o-core/src/main/java/hex/FeatureInteractions.java

```diff
@@ -43,6 +43,9 @@ public void mergeWith(FeatureInteractions featureInteractions) {
     }
 
     public int maxDepth() {
+        if (this.entrySet().size() == 0){
+            return 0;
```

Comment:
It's weird that actual problem is that FeatureInteractions are empty and all call has no meaning and we handle it in maxDepth funtion. It would be better to check that earlier and raise warning to the user.

Reply:
It can be simulated this way also:

```
FeatureInteractions featureInteractions1 = new FeatureInteractions();
System.out.println("featureInteractions1 = " + Arrays.toString(featureInteractions1.getAsTable()));
```

in `FeatureInteractionsTest`

Reply:
Thanks, @valenad1. Good points!  I fixed the case there is no feature interaction for a model.